test/test_dataset.py

```diff
@@ -24,25 +24,25 @@ def test_open_grid_only(self):
         uds2 = ux.open_dataset(uds2_name)
         uds3 = ux.open_dataset(uds3_name)
 
-        assert (uds1.nMesh2_node.size == constants.NNODES_ov_RLL10deg_CSne4)
-        assert (uds2.nMesh2_node.size == constants.NNODES_outCSne8)
-        assert (uds3.nMesh2_node.size == constants.NNODES_outCSne30)
+        assert (uds1.ds.nMesh2_node.size == constants.NNODES_ov_RLL10deg_CSne4)
```

Comment:
This use reminds me another discussion: Should we provide a direct `uds1.nMesh2_node` or is it sufficient to access it through `ds` like here, i.e. `uds1.nMesh2_node`?

Reply:
Right, we should not. That would be fixed in another PR, when we fix ugrid reader. It would need wholesale changes to all mentions of "Mesh2", in this case nMesh2_node. It'd be the `variable name` as specified in the original file.

Reply:
Sounds good!